src/components/character-controller.js

```diff
@@ -49,25 +36,7 @@ AFRAME.registerComponent("character-controller", {
 
   play: function() {
     const eventSrc = this.el.sceneEl;
-    eventSrc.addEventListener("stop_moving", this.onStopMoving);
-    eventSrc.addEventListener("translateX", this.onTranslateX);
-    eventSrc.addEventListener("translateY", this.onTranslateY);
-    eventSrc.addEventListener("translateZ", this.onTranslateZ);
-    eventSrc.addEventListener("action_move_forward", this.onMoveForward);
-    eventSrc.addEventListener(
-      "action_dont_move_forward",
-      this.onDontMoveForward
-    );
-    eventSrc.addEventListener("action_move_backward", this.onMoveBackward);
-    eventSrc.addEventListener(
-      "action_dont_move_backward",
-      this.onDontMoveBackward
-    );
-    eventSrc.addEventListener("action_move_left", this.onMoveLeft);
-    eventSrc.addEventListener("action_dont_move_left", this.onDontMoveLeft);
-    eventSrc.addEventListener("action_move_right", this.onMoveRight);
-    eventSrc.addEventListener("action_dont_move_right", this.onDontMoveRight);
-
+    eventSrc.addEventListener("move", this.onMove);
```

Comment:
IMO it's better for handler functions to have descriptive names instead of simply repeating the event name. 
e.g. This could be `eventSrc.addEventListener("move", this.setAccelerationInput);` instead.